### ISSUE TITLE
Specify what are you using version of; add info that you can set node version

### ISSUE THREAD

**Issue by WhyINeedToFillUsername** [satanTime/puppeteer-node#19]
1. in https://satantime.github.io/puppeteer-node/, at the section with pipelines, you have written
`image: satantime/puppeteer-node:16.15.1 # put here version you need` but you don't specify version of we can put there. I assume we can specify the version of node, and I tried it and it work. It would be very helpful to add it to the comment.

2. similarly, in https://hub.docker.com/r/satantime/puppeteer-node, you say it can provide all versions of node. But there is no information on how to do it. It would be very helpful to add that information there, something like "you can specify the node.js version by adding ":<node version> to the image name, e.g. `satantime/puppeteer-node:20`"

**Comment by satanTime** [comment]
Hi @WhyINeedToFillUsername,

could you check wether the updated examples help better?

**Comment by WhyINeedToFillUsername** [comment]
yes, that helps, thank you :) hopefully it'll will help others as well.
